shopfloor/services/checkout.py

```diff
@@ -352,22 +352,42 @@ def select(self, picking_id):
             return self._response_for_manual_selection(message=message)
         return self._select_picking(picking, "manual_selection")
 
-    def _select_lines(self, lines):
+    def _select_lines(self, lines, no_prefill_qty=None):
         for line in lines:
             if line.shopfloor_checkout_done:
                 continue
-            line.qty_done = line.product_uom_qty
+            if no_prefill_qty is None:
+                qty = line.product_uom_qty
+            else:
+                qty = no_prefill_qty
+            self._set_or_increment_qty_done(line, qty)
```

Comment:
`no_prefill_qty` seems to be a flag but is used below as a value.... I would rename it to `prefill_qty` and default it to -1, then
```suggestion
            qty = line.product_uom_qty if prefill_qty > 0 else prefill_qty
            self._set_or_increment_qty_done(line, qty)
```

Reply:
done